e2e/LongHaul/device/IotHub.cs

```diff
@@ -287,22 +289,25 @@ private Task<DirectMethodResponse> DirectMethodCallback(DirectMethodRequest meth
                                 C2dDirectMethodDelaySeconds,
                                 (DateTimeOffset.UtcNow - methodPayload.CurrentTimeUtc).TotalSeconds);
 
+                            // Set "methodCallsCount" obtained from the method payload as one of the reported properties.
+                            await SetPropertiesAsync("methodCallsCount", methodPayload.MethodCallsCount).ConfigureAwait(false);
```

Comment:
This isn't quite what I had in mind.

The service client will send a twin property request to the device. The device just needs to respond to property update requests and set them. This will be independent of direct methods working.

We'll also want a separate running task (like telemetry) that sets a twin property on a frequency.